Libraries/Modal/Modal.js

```diff
@@ -23,9 +21,7 @@ const View = require('../Components/View/View');
 const RCTModalHostView = require('./RCTModalHostViewNativeComponent');
 
 const ModalEventEmitter =
-  Platform.OS === 'ios' && NativeModules.ModalManager
-    ? new NativeEventEmitter(NativeModules.ModalManager)
-    : null;
+  Platform.OS === 'ios' ? require('./NativeModalManager').default : null;
```

Comment:
change this to an import, also, do the condition in `NativeModalManager.js` not here.